dpgen/generator/run.py

```diff
@@ -762,9 +777,9 @@ def run_train(iter_index, jdata, mdata):
         if training_init_model:
             init_flag = " --init-model old/model.ckpt"
         elif training_init_frozen_model is not None:
-            init_flag = " --init-frz-model old/init.pb"
+            init_flag = " --init-frz-model old/init%s" % suffix
         elif training_finetune_model is not None:
-            init_flag = " --finetune old/init.pb"
+            init_flag = " --finetune old/init%s" % suffix
         command = f"{train_command} train {train_input_file}{extra_flags}"
```

Comment:
For pt, we use `dp --pt train`

Reply:
I think this should be set from the key: 
```json
"train": [
        {
	        "command": "dp --pt"
        }
]
```

Reply:
This is okay, but needs documentation.